src/qibo/backends/__init__.py

```diff
@@ -41,7 +41,18 @@ def construct_backend(backend, platform=None, runcard=None):
             else:
                 platform = construct_backend(platform)
         return CliffordBackend(platform)
+    elif backend == "qibo":  # pragma: no cover
```

Comment:
Shouldn't we call it *cloud*, or something like?

`"qibo"` is not much informative (in this context...)

Reply:
I originally called them `qibo-client` and `qiskit-client` but then reverted them to just `qibo` and `qiskit`. Maybe `qibo-cloud` and `qiskit-cloud`?